### PR TITLE
Add conda-forge recipe for sos-sas

### PR DESCRIPTION
**Pull request by BoPeng**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [ x] Package does not vend other packages
- [ x] Build number is 0
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

NOTE: cannot really test because SAS is proprietary.

### CODE REVIEW

File: recipes/sos-sas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8611cb450f86f2d07b74ea1043ec0bc40fb1b19a98c8c7aa52db889b12211891
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```
Comment (marcelotrevisani) [inline recipes/sos-sas/meta.yaml]: Could it be ``noarch``?
```suggestion
    - {{ PYTHON }} -m pip install . -vv
  noarch: python
```

File: recipes/sos-sas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook >=0.19.4
+    - sas_kernel
+    - saspy
+    - sas7bdat
```
Comment (marcelotrevisani) [inline recipes/sos-sas/meta.yaml]: Please, add ``python`` here as well
if it is a ``noarch`` recipe you can do 
```suggestion
    - sas7bdat
    - python >=3.6
```

Comment (BoPeng) [reply recipes/sos-sas/meta.yaml]: Just did it.

**Comment by BoPeng** [comment]
@marcelotrevisani Windows and MacOSX tests fail because [`sas_kernel` is only available for Linux](https://anaconda.org/conda-forge/sas_kernel). I think the general idea is to leave `sos-sas` as `noarch: python` because `sas_kernel` could be available for these platforms later.

File: recipes/sos-sas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8611cb450f86f2d07b74ea1043ec0bc40fb1b19a98c8c7aa52db889b12211891
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+  noarch: python
```
Comment (scopatz) [inline recipes/sos-sas/meta.yaml]: you don't need noarch python twice :wink:

Comment (BoPeng) [reply recipes/sos-sas/meta.yaml]: Thanks. Not sure how it got there.

**Pull request merged**